mapproxy/config/loader.py

```diff
@@ -987,6 +987,15 @@ def _sqlite_cache(self, grid_conf, file_ext):
             cache_dir,
         )
 
+    def _mongodb_cache(self, grid_conf, file_ext):
+        from mapproxy.cache.mongodb import MongoDBCache
+
+        url = self.conf['cache'].get('url')
+        grid = grid_conf.conf['srs'].replace(':', '')
```

Comment:
You should use the grid name here:
https://github.com/ndamiens/mapproxy/blob/8575efcd14a11cd09c11b71b829dfb8c3f2ba139/mapproxy/config/loader.py#L1038..L1039